packages/cli/src/commands/configure/configure_command.ts

```diff
@@ -36,6 +36,7 @@ export class ConfigureCommand implements CommandModule<object> {
    */
   builder = (yargs: Argv): Argv => {
     return yargs
+      .version(false)
```

Comment:
Is there any way we can set this globally on the yargs parser?

Reply:
We decided it's ok to let the `--version` works for every command right? so just setting on the main_parser_factory is sufficient?

Reply:
We mentioned about having them returning the same version or disable the versions for sub-commands, but didn't have a hard conclusion.
IMO, having `amplify sandbox --version` return the same version as `amplify --version` might be confusing. It makes more sense to just disable `--version` for the sub-commands.

Reply:
I think `--version` doesn't make much sense on subcommands, but I don't think it matters much, so I'm fine either way, with preference of disabling this on subcommands.

Reply:
My preference is also disabling on subcommands. It's just noise in the help printouts

Reply:
I'm assuming the verdict on this is that it has to be set on each subcommand?

Reply:
If we want to hide it from subcommands then I believe yes.

Reply:
Yea, the `--version` option is hidden from `--help` now

Reply:
> Is there any way we can set this globally on the yargs parser?

Unfortunately, I was not able to find a way to set it globally from yargs. If we want, I guess we can customize a `amplifyYargs`. What do you think? @edwardfoyle

Reply:
Eh, probably not worth it at this point